cobrawap/pipeline/utils/neo_utils.py

```diff
@@ -84,7 +84,7 @@ def flip_image(imgseq, axis=-1):
             flipped_imgseq.annotations['array_annotations'][key] \
                         = np.flip(value, axis=axis)
         flipped_imgseq.annotations['array_annotations']['y_coords'] \
-                        = imgseq.annotations['array_annotations']['y_coords']
+                    = imgseq.annotations['array_annotations']['y_coords']
```

Comment:
This doesn't seem like a required change.

Reply:
It strengthen the smaller indentation of lines 86 and 87 with respect to the previous lines 84 and 85.